tests/test_operation_builder.py

```diff
@@ -166,14 +166,14 @@ def test_operand_builder_operation():
     op1 = ResultOp.build(result_types=[0])
     op2 = OperandOp.build(operands=[op1])
     op2.verify()
-    assert op2.operands == [op1.res]
+    assert op2.operands == (op1.res, )
```

Comment:
Can we take a moment to appreciate the source code here? Why do you have to add a trailing comma to get a single element tuple? Python please.

Reply:
Yep :) That's why a lot of languages make single-element tuples not expressible